tools/trace.py

```diff
@@ -20,6 +20,9 @@
 import traceback
 import sys
 
+if sys.version_info >= (3, 0):
+    xrange=range
```

Comment:
Why not just change xrange to range in the body of the file? Or, looking at those uses, they would be good candidates for enumerate().